client/src/components/FoodSeeker/SearchResults/ResultsMap/ResultsMap.js

```diff
@@ -204,21 +204,21 @@ const ResultsMap = (
           </Source>
         )}
       </Map>
-      {!loading && hasAdvancedFilterFeatureFlag && isMobile && (
+      {!loading && hasAdvancedFilterFeatureFlag && (
         <Grid
-          container
-          wrap="nowrap"
+          container={isMobile}
+          className={!isMobile ? "advanced-filters-class" : ""}
```

Comment:
what are we using this class for?

Reply:
Nice catch. Thanks Hannah, I think we can get rid of this, this class was with the deleted Grid component.